js/project-page-functions.js

```diff
@@ -332,8 +332,22 @@ function printAll(){
     case "test-door43.org.s3-website-us-west-2.amazonaws.com":
       api_prefix = "test-";
       break;
+    default:
```

Comment:
Won't this change mean that production will always get the the `dev-` prefix?

Reply:
Yes... this shouldn't have been pushed, will fix